internal/controller/health/checkers/builtin/argocd.go

```diff
@@ -54,46 +55,66 @@ type compositeError interface {
 	Unwrap() []error
 }
 
-// RunHealthCheckStep implements the Directive interface.
-func (a *argocdUpdater) RunHealthCheckStep(
+type argocdChecker struct {
```

Comment:
Arguably, the `Checker` suffix has become redundant.

Reply:
I knew there was an argument to be made for that since `checkers/` (which may become `checker/`) is in the import path.

If the package name had been `checker`, I'd have considered `checker.argocd` clear enough, but with the package name being `builtin`, I think `builtin.argocdChecker` is a bit more clear about what it is than `builtin.argocd` is.

I realize this is a bit academic because that type isn't even exported, so you're never going to see `builtin.argocdChecker` anywhere in the code. 🤷‍♂️ 

I'm willing to change it if you feel strongly.

Reply:
Going to leave this one be.